src/components/page-section.tsx

```diff
@@ -0,0 +1,25 @@
+import { Box } from 'theme-ui';
+
+export interface PageSectionProps {
+  backgroundColor?: 'green';
+}
+
+export const PageSection: React.FC<PageSectionProps> = ({
+  children,
+  backgroundColor: backgroundColorProp,
+}) => {
+  const backgroundColor =
+    backgroundColorProp === 'green' ? 'green100' : 'green20';
+  return (
+    <Box
+      sx={{
+        backgroundColor,
+        py: ['98px', '194px'],
```

Comment:
```suggestion
        py: ['6.125rem', '12.125rem'],
```

Maybe we can change these to `rem` values?

Reply:
Not sure why, but this change actually impacts the layout of the page... maybe we can take a look together?